proto/test/v1/proto3/test_all_types.proto

```diff
@@ -0,0 +1,108 @@
+syntax = "proto3";
+
+package google.api.expr.test.v1.proto3;
+
+import "google/protobuf/any.proto";
+import "google/protobuf/duration.proto";
+import "google/protobuf/struct.proto";
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/wrappers.proto";
+
+// This proto includes every type of field in both singular and repeated
+// forms.
+message TestAllTypes {
+  message NestedMessage {
+    // The field name "b" fails to compile in proto1 because it conflicts with
+    // a local variable named "b" in one of the generated methods.
+    // This file needs to compile in proto1 to test backwards-compatibility.
+    int32 bb = 1;
+  }
+
+  enum NestedEnum {
+    FOO = 0;
+    BAR = 1;
+    BAZ = 2;
+  }
+
+  // Singular
+  int32 single_int32 = 1;
+  int64 single_int64 = 2;
+  uint32 single_uint32 = 3;
+  uint64 single_uint64 = 4;
+  sint32 single_sint32 = 5;
+  sint64 single_sint64 = 6;
+  fixed32 single_fixed32 = 7;
+  fixed64 single_fixed64 = 8;
+  sfixed32 single_sfixed32 = 9;
+  sfixed64 single_sfixed64 = 10;
+  float single_float = 11;
+  double single_double = 12;
+  bool single_bool = 13;
+  string single_string = 14;
+  bytes single_bytes = 15;
+
+  // Wellknown.
+  google.protobuf.Any single_any = 100;
+  google.protobuf.Duration single_duration = 101;
+  google.protobuf.Timestamp single_timestamp = 102;
+  google.protobuf.Struct single_struct = 103;
+  google.protobuf.Value single_value = 104;
+  google.protobuf.Int64Value single_int64_wrapper = 105;
+  google.protobuf.Int32Value single_int32_wrapper = 106;
+  google.protobuf.DoubleValue single_double_wrapper = 107;
+  google.protobuf.FloatValue single_float_wrapper = 108;
+  google.protobuf.UInt64Value single_uint64_wrapper = 109;
+  google.protobuf.UInt32Value single_uint32_wrapper = 110;
+  google.protobuf.StringValue single_string_wrapper = 111;
+  google.protobuf.BoolValue single_bool_wrapper = 112;
+  google.protobuf.BytesValue single_bytes_wrapper = 113;
+
+  // Nested messages
+  oneof nested_type {
+    NestedMessage single_nested_message = 18;
+    NestedEnum single_nested_enum = 21;
+  }
+
+  // Repeated
+  repeated int32 repeated_int32 = 31;
+  repeated int64 repeated_int64 = 32;
+  repeated uint32 repeated_uint32 = 33;
+  repeated uint64 repeated_uint64 = 34;
+  repeated sint32 repeated_sint32 = 35;
+  repeated sint64 repeated_sint64 = 36;
+  repeated fixed32 repeated_fixed32 = 37;
+  repeated fixed64 repeated_fixed64 = 38;
+  repeated sfixed32 repeated_sfixed32 = 39;
+  repeated sfixed64 repeated_sfixed64 = 40;
+  repeated float repeated_float = 41;
+  repeated double repeated_double = 42;
+  repeated bool repeated_bool = 43;
+  repeated string repeated_string = 44;
+  repeated bytes repeated_bytes = 45;
+
+  repeated NestedMessage repeated_nested_message = 48;
+
```

Comment:
nit: Was this a field that needed to be removed? If so, you could renumber or alternatively reserve the field. In any event, consider grouping these seemingly related fields together.

Reply:
Added a comment and tightened up the grouping. Field numbering was inherited from cel-go/test/proto[23]pb/*.proto.  Updated it to batch by group.